zkevm-circuits/src/evm_circuit/execution/callop.rs

```diff
@@ -68,7 +72,8 @@ pub(crate) struct CallOpGadget<F> {
     precompile_return_length: Cell<F>,
     precompile_return_length_zero: IsZeroGadget<F>,
     return_data_copy_size: MinMaxGadget<F, N_BYTES_MEMORY_ADDRESS>,
-    input_bytes_rlc: Cell<F>,  // input bytes to precompile call.
+    input_len: Cell<F>, // the number of input bytes taken for the precompile call.
```

Comment:
it is only used for precompiles and should be 0 for non precompiles? so rename it with "precompile_" prefix to avoid mis understanding?